### PR TITLE
LPD-21765 Change logged-in user to the selected User

### PR DESCRIPTION
**Pull request by gergelyszalay**
Hi Team,

May I ask to review this PR?

Issue https://liferay.atlassian.net/browse/LPD-21765.

The root cause is in the GenerateWebDAVPasswordMVCActionCommand.java, we are getting the logged in userId, not the selected one.

Solution: added the p_u_i_d to the actionURL, and used it in GenerateWebDAVPasswordMVCActionCommand to get the selected user.

Locally tested.

Thank you in advance!
BR,

### CODE REVIEW

**Comment by gergelyszalay** [comment]
ci:test:relevant

**Comment by gergelyszalay** [comment]
ci:test:sf

File: modules/apps/users-admin/users-admin-web/src/main/resources/META-INF/resources/user/generate_webdav_password.jsp [hunk 1]
```diff
@@ -7,10 +7,14 @@
 
 <%@ include file="/init.jsp" %>
 
+<%
+User selUser = userDisplayContext.getSelectedUser();
```
Comment (smottal) [inline modules/apps/users-admin/users-admin-web/src/main/resources/META-INF/resources/user/generate_webdav_password.jsp:11]: please rename this in `User selectedUser = ` if not already used

**Review by smottal — CHANGES_REQUESTED** [review]

**Comment by smottal** [comment]
@gergelyszalay just a very little change. Thx

**Review by smottal — APPROVED** [review]

**Comment by smottal** [comment]
ci:forward

**Comment by smottal** [comment]
ci:forward:force

**Pull request closed without merging**